packages/models/src/crypto-assets/crypto-asset-info.model.ts

```diff
@@ -0,0 +1,75 @@
+import { InscriptionMimeType } from './bitcoin/inscription.model';
+
+export interface CryptoAssetInfo {
+  readonly decimals: number;
+  readonly hasMemo: boolean;
+}
+
+export interface BtcCryptoAssetInfo extends CryptoAssetInfo {
+  readonly name: 'bitcoin';
+  readonly symbol: 'BTC';
+}
+
+export interface StxCryptoAssetInfo extends CryptoAssetInfo {
+  readonly name: 'stacks';
+  readonly symbol: 'STX';
+}
+
+/**
+ * Bitcoin
+ */
```

Comment:
This comment now shows only for `Brc20CryptoAssetInfo`. 

If it's a "section" comment, better to use a `//` comment